Airports/EGPK/VRPs.txt

```diff
@@ -1,11 +1,10 @@
-*A70 Cumnock Rbt:N055.27.14.000:W004.17.03.000
-*A76 Jct (Mauchline):N055.30.57.000:W004.22.45.000
-*B741 Dalmellington Rbt:N055.19.24.000:W004.23.50.000
-*B742 Jct (Dalrymple):N055.23.50.000:W004.35.33.000
-*Culzean Bay/Castle:N055.22.10.000:W004.46.05.000
+*Cumnock A70 Rbt:N055.27.14.000:W004.17.03.000
+*Mauchline A76 Jct:N055.30.57.000:W004.22.45.000
+*Dalmellington B741 Rbt:N055.19.24.000:W004.23.50.000
+*Dalrymple B742 Jct:N055.23.50.000:W004.35.33.000
+*Culzean Bay:N055.22.10.000:W004.46.05.000
 *Heads of Ayr:N055.25.58.000:W004.42.47.000
 *Irvine Harbour:N055.36.39.000:W004.41.55.000
 *Kilmarnock Rail Stn:N055.36.45.000:W004.29.54.000
-*Millennium Footbridge:N055.26.25.000:W004.39.03.000
-*Pladda:N055.25.35.000:W005.07.04.000
```

Comment:
I'm confused why @AliceFord said to remove this, since it's still in the VRP list for next AIRAC 
![image](https://github.com/VATSIM-UK/UK-Sector-File/assets/61326713/80b152de-a842-4a9e-a7e2-094d0f8ac3ac)

Reply:
It was because it's not on the chart - any ideas why? We can add it back, maybe we should ask NATA?

Reply:
I think it's just west of the chart limit, so yeah I would add it back, no need to contact NATS I don't think haha!

Reply:
I sorted this file alphabetically and re-added Pladda VRP, as requested.

Regarding the changelog file: I can directly add to the proper section (in this case below "changes from release 2023/09a to 2023/10"), if you mean this, but I guess the correct numbering must be done on final validation & merging with the main branch?

Reply:
Yeah, if you just add it below rather than at the top, and with the next number, it will then conflict with the other changes so we can quickly change it when merging. If you add the entry at the top with a `#.`, it doesn't conflict and we have to manually edit things :)